money-api/src/main/java/com/comcast/money/api/Span.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2012-2015 Comcast Cable Communications Management, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.comcast.money.api;
+
+/**
+ * A Span is a container that represents a single invocation of a traced service / operation.
```

Comment:
not necessarily an invocation.

A Span a respressentation of a basic unit of work.  Could be a method invocation, a single operation or a group of operations.  Offers a place to collect Notes describing the span.